.changeset/olive-maps-accept.md

```diff
@@ -0,0 +1,11 @@
+---
+'@spectrum-web-components/button': patch
```

Comment:
@Rajdeepc can you update this changeset to reflect the change to the dev.warning and remove the callout for css changes? those are captured in the mod fix changeset